pkg/hostagent/hostagent.go

```diff
@@ -243,6 +244,15 @@ func logPipeRoutine(r io.Reader, header string) {
 	}
 }
 
+func randomPassword(length int) string {
+	passwd := ""
+	rand.Seed(time.Now().UnixNano())
+	for i := 0; i < length; i++ {
+		passwd += strconv.Itoa(rand.Intn(10))
```

Comment:
numeric-only password is insecure

Reply:
the password is sent in clear-text

Reply:
Still we have to choose a secure password, [even for localhost](https://theevilbit.github.io/posts/vmware_fusion_11_guest_vm_rce_cve-2019-5514/)

Reply:
okay, I will try to find a suitable library and use that instead

Reply:
Added https://github.com/sethvargo/go-password